### PR TITLE
[IMP] portal_sale_distributor: change message when a sale is not conf…

### PR DESCRIPTION
**Pull request by lav-adhoc**
…irmed

### CODE REVIEW

**Comment by roboadhoc** [comment]
[Pull request status dashboard](https://mergebot.adhoc.ar/ingadhoc/sale/pull/754).

File: portal_sale_distributor/models/sale_order.py [hunk 1]
```diff
@@ -15,11 +15,17 @@ class SaleOrder(models.Model):
...
     def action_confirm_distributor(self):
-        self.sudo().message_post(
-            body=_("Pedido confirmado por %s") % self.env.user.name,
+        if self.detect_exceptions() != []:
+            self.sudo().message_post(
+            body=_("El pedido no puede ser confirmado por %s porque contiene excepciones, debe ser revisado por un administrador") % self.env.user.name,
             subtype_id=self.env.ref('mail.mt_comment').id)
-        self = self.sudo()
-        return self.action_confirm()
+            self = self.sudo()
```
Comment (lav-adhoc) [inline portal_sale_distributor/models/sale_order.py]: borrar esta linea

File: portal_sale_distributor_sales_team/i18n/es.po
Comment (lav-adhoc) [inline portal_sale_distributor_sales_team/i18n/es.po:1]: Eliminar archivo y carpeta

File: portal_sale_distributor_sales_team/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+{
+    'name': 'Portal Distributor MRP',
```
Comment (lav-adhoc) [inline portal_sale_distributor_sales_team/__manifest__.py]: cambiar nombre al modulo

File: portal_sale_distributor_sales_team/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    'category': 'Tools',
+    'complexity': 'easy',
+    'author': 'ADHOC SA',
+    'website': 'www.adhoc.com.ar',
+    'license': 'AGPL-3',
+    'depends': [
+        'sales_team',
+        'portal_sale_distributor',
+    ],
+    'demo': [
```
Comment (lav-adhoc) [inline portal_sale_distributor_sales_team/__manifest__.py]: borrar demo

File: portal_sale_distributor_product_pack/__manifest__.py [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+    'category': 'Tools',
+    'complexity': 'easy',
+    'author': 'ADHOC SA',
+    'website': 'www.adhoc.com.ar',
+    'license': 'AGPL-3',
+    'depends': [
+        'product_pack',
+        'portal_sale_distributor',
+    ],
+    'demo': [
```
Comment (lav-adhoc) [inline portal_sale_distributor_product_pack/__manifest__.py]: borrar demo

**Comment by matiasperalta1** [comment]
@roboadhoc rebase-ff

**Comment by roboadhoc** [comment]
Merge method set to rebase and fast-forward.

**Comment by jok-adhoc** [comment]
@roboadhoc rebase-ff

**Comment by roboadhoc** [comment]
Merge method set to rebase and fast-forward.

**Comment by jok-adhoc** [comment]
@roboadhoc r+

**Comment by fw-bot-adhoc** [comment]
@lav-adhoc @jok-adhoc this pull request has forward-port PRs awaiting action (not merged or closed):
- ingadhoc/sale#791

**Comment by fw-bot-adhoc** [comment]
@lav-adhoc @jok-adhoc this pull request has forward-port PRs awaiting action (not merged or closed):
- ingadhoc/sale#791

**Comment by fw-bot-adhoc** [comment]
@lav-adhoc @jok-adhoc this pull request has forward-port PRs awaiting action (not merged or closed):
- ingadhoc/sale#791

**Comment by fw-bot-adhoc** [comment]
@lav-adhoc @jok-adhoc this pull request has forward-port PRs awaiting action (not merged or closed):
- ingadhoc/sale#791

**Comment by fw-bot-adhoc** [comment]
@lav-adhoc @jok-adhoc this pull request has forward-port PRs awaiting action (not merged or closed):
- ingadhoc/sale#791

**Comment by fw-bot-adhoc** [comment]
@lav-adhoc @jok-adhoc this pull request has forward-port PRs awaiting action (not merged or closed):
- ingadhoc/sale#791

**Comment by fw-bot-adhoc** [comment]
@lav-adhoc @jok-adhoc this pull request has forward-port PRs awaiting action (not merged or closed):
- ingadhoc/sale#791

**Comment by fw-bot-adhoc** [comment]
@lav-adhoc @jok-adhoc this pull request has forward-port PRs awaiting action (not merged or closed):
- ingadhoc/sale#791

**Comment by fw-bot-adhoc** [comment]
@lav-adhoc @jok-adhoc this pull request has forward-port PRs awaiting action (not merged or closed):
- ingadhoc/sale#791

**Comment by fw-bot-adhoc** [comment]
@lav-adhoc @jok-adhoc this pull request has forward-port PRs awaiting action (not merged or closed):
- ingadhoc/sale#791

**Pull request closed without merging**